templates/resources/components/footer.md.tmpl

```diff
@@ -13,4 +13,14 @@ $ terraform import fastly_service_v1.demo xxxxxxxxxxxxxxxxxxxx
 ```{{end}}
 {{ if eq .Data.ServiceType "wasm"}}```
 $ terraform import fastly_service_compute.demo xxxxxxxxxxxxxxxxxxxx
+```{{end}}
+
+By default, either the active version will be imported, or the latest version if no version is active.
+Alternatively, a specific version of the service can be selected by appending an `@` followed by the version number to the service ID, e.g.
+
+{{ if eq .Data.ServiceType "vcl"}}```
+$ terraform import fastly_service_v1.demo xxxxxxxxxxxxxxxxxxxx@2
```

Comment:
Oh interesting. How is `xxxxxxxxxxxxxxxxxxxx` replaced with the mock Service ID?